account_payment_mode/tests/test_account_payment_mode.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2016-2020 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo.exceptions import UserError, ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountPaymentMode(TransactionCase):
+    def setUp(self):
```

Comment:
pls move this setup code to `setUpClass` and start like this:

```py
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Reply:
:+1: